### DISCUSSION TITLE
Update the Firmware

### DISCUSSION THREAD

**Discussion by nekton2** [ElectroTechnique/TSynth-Teensy4.1#127] [T4.1 code development]
Hello, the Build Guide says: "You usually need to 
press the button on the Teensy to flash the code onto it"

Is this necessary for every update?  When the case is closed, it's hard to press the button, isn't it?  Or through a hole in the bottom with a thin stick?

**Comment by wilksy** [comment]
Never had to press the button after the First flash. I just use the usb.

(Windows)

**Reply by nekton2** [reply]
But how? There is an extra hole in the base plate. For the button on the Teensy!
